modules/ztp-image-based-upgrade-talm.adoc

```diff
@@ -0,0 +1,375 @@
+  sourceFiles:
+    - fileName: SriovNetwork.yaml
+      policyName: "config-policy"
+      metadata:
+        name: "sriov-nw-du-fh"
+        labels:
+          lca.openshift.io/target-ocp-version: “4.16.0” <1>
+      spec:
+        resourceName: du_fh
+        vlan: 140
+    - fileName: SriovNetworkNodePolicy.yaml
+      policyName: "config-policy"
+      metadata:
+        name: "sriov-nnp-du-fh"
+        labels:
+          lca.openshift.io/target-ocp-version: “4.16.0” <1>
+      spec:
+        deviceType: netdevice
+        isRdma: false
+        nicSelector:
+          pfNames: ["ens5f0"]
+        numVfs: 8
+        priority: 10
+        resourceName: du_fh
+    - fileName: SriovNetwork.yaml
+      policyName: "config-policy"
+      metadata:
+        name: "sriov-nw-du-mh"
+        labels:
+          lca.openshift.io/target-ocp-version: “4.16.0” <1>
+      spec:
+        resourceName: du_mh
+        vlan: 150
+    - fileName: SriovNetworkNodePolicy.yaml
+      policyName: "config-policy"
+      metadata:
+        name: "sriov-nnp-du-mh"
+        labels:
+          lca.openshift.io/target-ocp-version: “4.16.0” <1>
+      spec:
+        deviceType: vfio-pci
+        isRdma: false
+        nicSelector:
+          pfNames: ["ens7f0"]
+        numVfs: 8
+        priority: 10
+        resourceName: du_mh
+----
+<1> Ensure that the `lca.openshift.io/target-ocp-version` label matches the target {product-title} version that is specified in the `ImageBasedUpgrade` CR. The {lcao} only apply the CRs that match the version specified in the `seedImageRef.version` field.
+
+. Create the `Upgrade` stage `ImageBasedUpgrade` CR policy with the OADP `ConfigMap` reference:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: group-ibu
+  namespace: "ztp-group"
+spec:
+  bindingRules:
+    group-du-sno: ""
+  mcp: "master"
+  sourceFiles:
+    - fileName: ImageBasedUpgrade.yaml
+      policyName: "upgrade-policy"
+      metadata:
+        name: "upgrade"
+      spec:
+        stage: Upgrade
+        oadpContent:
+        - name: "oadp-cm"
+          namespace: "openshift-adp"
+        seedImageRef:
+          version: "4.16.0"
+          image: "quay.io/user/upgbackup:lca-seed"
+          pullSecretRef:
+            name: "seed-pull-secret"
+      status:
+        conditions:
+          - reason: Completed
+            status: "True"
+            type: UpgradeCompleted
+----
+
+. Create the `ClusterGroupUpgrade` CR that references the `ImageBasedUpgrade` CR with the `Upgrade` stage:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-ibu-upgrade
+  namespace: default
+spec:
+  clusters:
+  - spoke1
+  enable: true
+  managedPolicies:
+  - group-ibu-upgrade-policy
+  remediationStrategy:
+    canaries:
+      - spoke1
+    maxConcurrency: 1
+    timeout: 240
+----
+
+. To commit to the changes, create the following `ImageBasedUpgrade` policy:
```

Comment:
I dont understand the sentence "To commit to chantes". Following the previous steps would be?: "Back to the "idle" stage again, create the following..."

Reply:
I'm not sure I understand the question but probably the `commit to the changes` is confusing. I could add something like what we have in the [Upgrading](https://69809--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies#ztp-image-based-upgrading-with-backup_ztp-talm) section:
> Once you are satisfied with the upgrade, commit to the change by editing the value of the stage field to Idle in the ImageBasedUpgrade CR:

Is this what you are looking for?

Reply:
In line 210, you have already done the upgrade?. So now, you want to set the stage to 'Idle' to mark it as upgraded finished, no?
So the sentence should be something about it: "Once you are satisfied with the upgrade, you have to set back the cluster to Idle stage". Create the following `ImageBasedUpgrade` policy.